WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailCoordinator.swift

```diff
@@ -447,15 +451,18 @@ extension ReaderDetailCoordinator: ReaderDetailHeaderViewDelegate {
         previewSite()
     }
 
-    func didTapFeaturedImage(_ sender: CachedAnimatedImageView) {
-        showFeaturedImage(sender)
-    }
-
     func didTapFollowButton() {
         followSite()
     }
 }
 
+// MARK: - ReaderDetailFeaturedImageViewDelegate
+extension ReaderDetailCoordinator: ReaderDetailFeaturedImageViewDelegate {
+    func didTapFeaturedImage(_ sender: CachedAnimatedImageView) {
+        showFeaturedImage(sender)
```

Comment:
When I tap an image and then go back, I can see the nav bar blue and then this disappears. Is this something that you're aware of?

We can address that later, anyway.

Reply:
yeah ☹️ I spent a few hours working on this but could not come up with a solution. For some reason the appearance is reset when the image view is opened, and doesn't reset itself in viewWillAppear, so I have to reset the appearance again in viewDidAppear.